lib/rules/block-opening-brace-newline-after/index.js

```diff
@@ -29,13 +30,22 @@ function rule(expectation, options, context) {
 			possible: ['always', 'always-multi-line', 'never-multi-line'],
 		});
 
-		if (!validOptions) {
+		if (
```

Comment:
We should update the schema on line `30` instead of changing this conditional. Look at other rules with `ignore` secondary options do this.